### PR TITLE
Wipe info revamp

### PR DESCRIPTION
**Pull request by ozgunozerk**
This PR:
- revamps `wipe` and `info` command w.r.t `summary` changes
- adds tokio console tracing

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -56,12 +56,7 @@ enum Commands {
         executor: bool,
     },
     #[command(about = "wipes the node and farm instance (along with your plots)")]
-    Wipe {
-        #[arg(long, action)]
-        farmer: bool,
-        #[arg(long, action)]
-        node: bool,
```
Comment (i1i1) [inline src/main.rs:63]: I think you shouldn't remove flags. I think the path should be as follow:
```
if terminal && !farmer && !node {
  interactive_prompts();
} else {
  current_code();
}
```

Comment (ozgunozerk) [reply src/main.rs:63]: will think about that, thanks!

Comment (i1i1) [reply src/main.rs:63]: `terminal` bool here should be check whether stdout is terminal. This crate will do the job I think: https://crates.io/crates/is-terminal

**Comment by ozgunozerk** [comment]
I'll spawn tokio tasks with names in the future. I might have other stuff for optimizing the debugging, so saving it for a later pr

**Comment by ozgunozerk** [comment]
closed in favor of #166 
i don't want to edit cargo.lock file manually :)

**Pull request closed without merging**